worker/scripts/getmake.py

```diff
@@ -1,15 +1,17 @@
-import io, tarfile, urllib.request
+import io, hashlib, tarfile, urllib.request
 
-def get(url):
-  tar = tarfile.open(fileobj=io.BytesIO(urllib.request.urlopen(url).read()))
+def get(url, sha1):
```

Comment:
I'd strongly prefer sha256 though, sha1 shouldn't really be used for anything security-related anymore

Reply:
No problem, sha1 is just what happens to be in the files info at Sourceforge. Changed the check to sha256.